### PR TITLE
Code length reduction

### PR DESCRIPTION
**Pull request by smartcrash**
Reduce balloon.min.css bundled file size from ~8.1KB to ~6.6KB preserving the same appearance and interface.

### CODE REVIEW

File: src/_mixins.scss [hunk 1]
```diff
@@ -1,29 +1,19 @@
...
-
-@mixin arrow-left() {
-  width: 0;
-  height: 0;
-  border: $balloon-arrow-size solid transparent;
-  border-right-color: var(--balloon-color);
-}
+@mixin arrow($direction) {
+  @if not map-has-key($ARROW_DIRECTIONS, $direction) {
+    @error '[balloon] The $direction parameter most be one of: `#{map-keys($DIRECTIONS)}, `#{$direction}` was given.';
```
Comment (kazzkiq) [inline src/_mixins.scss]: Small typo, `most` should be `must`.

**Review by kazzkiq — CHANGES_REQUESTED** [review]
Pretty good overall refactoring. I see no issues with the code other than the typo. Will run some manual tests to confirm the behavior stays the same (especially on mobile).

**Comment by smartcrash** [comment]
I removed the typo 👌

**Comment by kazzkiq** [comment]
Finally got time to do some regressive testing and everything seems ok.

That's a great contribution. Thanks!

**Pull request merged**